src/main/java/sandbox/SqlHelper.java

```diff
@@ -0,0 +1,431 @@
+package sandbox;
+
+import org.commcare.modern.util.Pair;
+import org.commcare.modern.database.*;
+import org.javarosa.core.services.Logger;
+import org.javarosa.core.services.storage.Persistable;
+
+import java.io.ByteArrayInputStream;
+import java.sql.Blob;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Set of Sql utility methods for clients running modern, non-Android Java (where prepared
+ * statements in the place of cursors)
+ * <p/>
+ * All methods that return a ResultSet expect a PreparedStatement as an argument and the caller
+ * is responsible for closing this statement when its finished with it.
+ * <p/>
+ * Created by wpride1 on 8/11/15.
+ */
+public class SqlHelper {
+
+    public static final boolean SQL_DEBUG = true;
+
+    public static void dropTable(Connection c, String storageKey) {
+        String sqlStatement = "DROP TABLE IF EXISTS " + storageKey;
+        PreparedStatement preparedStatement = null;
+        try {
+            preparedStatement = c.prepareStatement(sqlStatement);
+            preparedStatement.execute();
+        } catch (SQLException e) {
+            Logger.log("E", "Could not drop table: " + e.getMessage());
+        } finally {
+            if (preparedStatement != null) {
+                try {
+                    preparedStatement.close();
+                } catch (SQLException e) {
+                    throw new RuntimeException(e);
+                }
+            }
+        }
```

Comment:
That's a great idea... didn't even think of that becoming available to us by moving into Formplayer repository. I think that's a bit higher touch, though, since we currently rely on accessing the Connection for managing the transaction, so I might leave until a separate PR